src/lib/hooks/useRoadmap.ts

```
@@ -0,0 +1,386 @@
+        // Revert optimistic update on error
+        setSuggestions(prev =>
+          prev.map(s =>
+            s.id === suggestionId
+              ? { ...s, votes_count: s.votes_count - 1, hasUserVoted: false }
+              : s
+          )
+        );
+        throw new Error(error.message);
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)